pkg/kubelet/metrics/metrics.go

```diff
@@ -736,6 +737,17 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	// ContainerPostStartHookDuration is the duration of container's PostStart hook.
+	ContainerPostStartHookDuration = metrics.NewHistogram(
```

Comment:
I feel like without a pod label this metric will be very difficult to use since post start hooks may be completely different from pods to pods. I can hardly see how we could make an alert out of this.

Do we perhaps that information already exposed somewhere like in the pod status for example?